Lib/test/test_isinstance.py

```diff
@@ -247,15 +238,8 @@ def test_subclass_tuple(self):
         self.assertEqual(False, issubclass(Child, ()))
         self.assertEqual(True, issubclass(Super, (Child, (Super,))))
 
-        self.assertEqual(True, issubclass(NewChild, (NewChild,)))
-        self.assertEqual(True, issubclass(NewChild, (NewSuper,)))
-        self.assertEqual(False, issubclass(NewSuper, (NewChild,)))
-        self.assertEqual(True, issubclass(NewSuper, (NewChild, NewSuper)))
-        self.assertEqual(False, issubclass(NewChild, ()))
-        self.assertEqual(True, issubclass(NewSuper, (NewChild, (NewSuper,))))
-
         self.assertEqual(True, issubclass(int, (int, (float, int))))
-        self.assertEqual(True, issubclass(str, (str, (Child, NewChild, str))))
+        self.assertEqual(True, issubclass(str, (str, (Child, str))))
```

Comment:
Wasn't 100% what to do here so I simply chose to remove `NewChild` and keep the test.